cmake_modules/ThirdpartyToolchain.cmake

```diff
@@ -197,6 +203,13 @@ if (ORC_PACKAGE_KIND STREQUAL "conan")
   add_resolved_library (orc_zlib ${ZLIB_LIBRARIES} ${ZLIB_INCLUDE_DIR})
   list (APPEND ORC_SYSTEM_DEPENDENCIES ZLIB)
   list (APPEND ORC_INSTALL_INTERFACE_TARGETS "$<INSTALL_INTERFACE:ZLIB::ZLIB>")
+elseif (ORC_PACKAGE_KIND STREQUAL "vcpkg")
+  find_package(ZLIB REQUIRED)
+  add_library (orc_zlib INTERFACE IMPORTED)
+  target_link_libraries(orc_zlib INTERFACE ZLIB::ZLIB)
+  list (APPEND ORC_VCPKG_DEPENDENCIES ZLIB)
```

Comment:
Should this line be deleted since `ORC_VCPKG_DEPENDENCIES` is not used elsewhere?

Reply:
Good catch, thanks @wgtmac